src/main/java/com/github/onsdigital/babbage/configuration/Configuration.java

```diff
@@ -140,6 +156,35 @@ public static Integer getElasticSearchPort() {
         public static String getElasticSearchCluster() {
             return elasticSearchCluster;
         }
+
+        /**
+         * Method to load the list of retired product pages to be hidden
+         * @return List of url strings containing the black listed urls
+         */
+        private static List<String> loadHighlightBlacklist() {
+            ClassLoader classLoader = Configuration.class.getClassLoader();
+            URL fileUrl = classLoader.getResource(HIGHLIGHT_URL_BLACKLIST_FILE);
+
+            List<String> urls = new ArrayList<>();
+            if (null != fileUrl) {
+                File file = new File(fileUrl.getFile());
+                try (BufferedReader bw = new BufferedReader(new FileReader(file))) {
+                    String blacklistedUrl;
+
+                    while ((blacklistedUrl = bw.readLine()) != null) {
+                        urls.add(URIUtil.cleanUri(blacklistedUrl));
+                    }
+                } catch (IOException e) {
+                    // Print additional info out to stderr
+                    String message = "Error while attempting to load highlight blacklist file.";
+                    System.out.println(message);
```

Comment:
we've started using different style of logging elsewhere, we should probably think about updating this

e.g.
https://github.com/ONSdigital/babbage/blob/1c94598ca1c8423ad82d44a5d06611795cfb7ebf/src/main/java/com/github/onsdigital/babbage/template/handlebars/helpers/markdown/util/ChartTagReplacer.java#L48

Reply:
there's also a few other uses of `System.out.println` in this PR